package.json

```diff
@@ -180,7 +180,8 @@
     "node-sass": "^5.0.0",
     "playwright-core": "^1.9.1",
     "playwright-electron": "^0.5.0",
-    "postcss-loader": "^3.0.0",
+    "postcss": "^8.3.0",
```

Comment:
The breaking change of `postcss-loader` moved `postcss` to a peer dependency, that is why I added it here.